routers/web/explore/user.go

```diff
@@ -132,8 +139,16 @@ func Users(ctx *context.Context) {
 	ctx.Data["PageIsExploreUsers"] = true
 	ctx.Data["IsRepoIndexerEnabled"] = setting.Indexer.RepoIndexerEnabled
 
-	if ctx.FormString("sort") == "" {
-		ctx.SetFormString("sort", setting.UI.ExploreDefaultSort)
+	supportedSortOrders := container.SetOf(
+		"newest",
+		"oldest",
+		"alphabetically",
+		"reversealphabetically",
+	)
```

Comment:
Can we perhaps extract that into a global constant instead of a local constant for maintainability?

Reply:
Well, there used to be, but it proves that it is an over-design.

![image](https://github.com/go-gitea/gitea/assets/2114189/4ecb181b-a42b-424c-8acd-4a6e84d9a87f)